swag/db.py

```diff
@@ -85,24 +108,64 @@ def add_user(self, user, guild):
         else:
             raise AccountAlreadyExist
 
+    def add_cagnotte(self, cagnotte_name, cagnotte_currency, cagnotte_creator_id):
+        if cagnotte_name not in self.cagnotte_id:
+            self.cagnotte_id[cagnotte_name] = self.next_cagnotte_id
+            self.next_cagnotte_id += 1
+
+            self.cagnotte_name.append(cagnotte_name)
+            if cagnotte_currency == Currency.SWAG:
+                self.cagnotte_balance.append(0)
+            elif cagnotte_currency == Currency.STYLE:
+                self.cagnotte_balance.append(Decimal(0))
+
+            self.cagnotte_currency.append(cagnotte_currency)
+            self.cagnotte_manager.append([cagnotte_creator_id])
+            self.cagnotte_participant.append(set())
+            self.cagnotte_activation.append(True)
+
+        else:
+            raise CagnotteNameAlreadyExist
+
     def get_account(self, user):
         try:
             return SwagAccount(self, self.id[user])
         except KeyError:
             raise NoAccountRegistered(user)
 
+    def get_cagnotte(self, cagnotte_name):
```

Comment:
Same comment, is this function relevant? Or should `get_cagnotte_from_index` be renamed to `get_cagnotte`?

Reply:
Resolved here https://github.com/BobbySoftwares/Bybbot/pull/7/commits/50df69e8dbcb5935bd5849b53157b42661ba1c65 and here https://github.com/BobbySoftwares/Bybbot/pull/7/commits/8455c48be74193cc6dda0089c51fecbed020817c